deploy/replicationcrds.all.yaml

```diff
@@ -1,9 +1,94 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
Only 'Install using repctl' option uses the deploy/*crd file. There is a similar file for Helm installation at helm/csm-replication/crds. Please update that one too with the changes here.

Reply:
Temporarily 2 CRD files are maintained at _scripts/upgrade/_ which are used for Replication module **upgrade**.
It was duplicated since a new _init_ image was created directly out of the files & scripts from there.